app/models/identifier_scheme.rb

```diff
@@ -38,9 +42,21 @@ class IdentifierScheme < ActiveRecord::Base
   # ===========================
 
   scope :active, -> { where(active: true) }
-  scope :for_users, -> { active.where(name: %w[shibboleth orcid]) }
-  scope :for_orgs, -> { active.where(name: %w[shibboleth ror fundref]) }
-  scope :for_plans, -> { active.where(name: %w[doi]) }
-  scope :authenticatable, -> { active.where(name: %w[shibboleth orcid]) }
+  scope :for_users, -> { active.where(for_users: true) }
+  scope :for_orgs, -> { active.where(for_orgs: true) }
+  scope :for_plans, -> { active.where(for_plans: true) }
+  scope :authenticatable, -> { active.where(for_auth: true) }
```

Comment:
I forget if these field names were needed for the model polymorphism.
If not, we might want to condiser bitflags (if not mutually exclusive), or an enum (if they are)